haas_websocket/cloud_functions/api_data_CF.py

```diff
@@ -54,24 +59,46 @@ def createDatastoreTasks(client, list, kind):
     else:
         return None
 
-# Entry point for cloud function
-def entry(request):
-    logging.basicConfig(format='%(levelname)s:%(message)s', level=logging.INFO)
-    rest_agent = TokenAuth()
+def parseApiData(rest_agent):
     rest_agent.signIn()
     organizations = rest_agent.getOrganizations()
+    print(rest_agent)
```

Comment:
Rather than print, should we use the logging framework?

Reply:
Didn't mean to leave this in, I am just going to delete it.